tests/pipeline_operations_test.py

```diff
@@ -323,14 +331,6 @@ def test_local_flat_map(self):
             [("a", 1), ("a", 2), ("a", 3), ("a", 4), ("b", 5), ("b", 6),
              ("b", 7), ("b", 8)])
 
-    def test_local_group_by_key(self):
```

Comment:
It looks like this was accidental removed

Reply:
I removed it because I thought I accidentally duplicated it. I realize in main right now it's on lines 248 and 379. Is it okay to remove one of the duplicates here or should that be its own pull request?

Reply:
Thanks a lot for catching this! Yeah, this PR is fine